src/main/resources/migration/changelog-v5.5.0.xml

```diff
@@ -170,4 +170,29 @@
         <addDefaultValue tableName="VULNERABILITY_POLICY" columnName="OPERATION_MODE" defaultValue="APPLY"/>
         <addNotNullConstraint tableName="VULNERABILITY_POLICY" columnName="OPERATION_MODE"/>
     </changeSet>
+
+    <changeSet id="v5.5.0-15" author="sahibamittal">
```

Comment:
Since this would be for the `5.6.0` release, consider moving this to a new changelog file as per https://dependencytrack.github.io/hyades/latest/development/database-migrations/#adding-migrations